kuma/wiki/feeds.py

```diff
@@ -141,8 +141,8 @@ def write(self, outfile, encoding):
             else:
                 revision = document
 
-            if revision.creator.email:
-                item_out['author_avatar'] = gravatar_url(revision.creator.email)
+            if revision.creator:
+                item_out['author_avatar'] = get_avatar_url(revision.creator)
```

Comment:
Because of this line, with `django-querycount` enabled, this happens:

```
web_1            | http://wiki.mdn.localhost:8000/en-US/docs/feeds/json/all/
web_1            | |------|-----------|----------|----------|----------|------------|
web_1            | | Type | Database  |   Reads  |  Writes  |  Totals  | Duplicates |
web_1            | |------|-----------|----------|----------|----------|------------|
web_1            | | RESP |  default  |   505    |    0     |   505    |     65     |
web_1            | |------|-----------|----------|----------|----------|------------|
web_1            | Total queries: 505 in 2.8313s
web_1            |
web_1            |
web_1            | Repeated 65 times.
web_1            | SELECT `socialaccount_socialaccount`.`id`,
web_1            | `socialaccount_socialaccount`.`user_id`,
web_1            | `socialaccount_socialaccount`.`provider`,
web_1            | `socialaccount_socialaccount`.`uid`,
web_1            | `socialaccount_socialaccount`.`last_login`,
web_1            | `socialaccount_socialaccount`.`date_joined`,
web_1            | `socialaccount_socialaccount`.`extra_data` FROM
web_1            | `socialaccount_socialaccount` WHERE
web_1            | (`socialaccount_socialaccount`.`user_id` = 9 AND
web_1            | `socialaccount_socialaccount`.`provider` = 'github')
web_1            |
web_1            | Repeated 51 times.
web_1            | SELECT `socialaccount_socialaccount`.`id`,
web_1            | `socialaccount_socialaccount`.`user_id`,
web_1            | `socialaccount_socialaccount`.`provider`,
web_1            | `socialaccount_socialaccount`.`uid`,
web_1            | `socialaccount_socialaccount`.`last_login`,
web_1            | `socialaccount_socialaccount`.`date_joined`,
web_1            | `socialaccount_socialaccount`.`extra_data` FROM
web_1            | `socialaccount_socialaccount` WHERE
web_1            | (`socialaccount_socialaccount`.`user_id` = 853 AND
web_1            | `socialaccount_socialaccount`.`provider` = 'github')
```

See https://github.com/escattone/kuma/pull/3

Reply:
Nice analysis and good idea, thanks @peterbe !

Reply:
Merged https://github.com/escattone/kuma/pull/3